webroot/js/directives/sentence-and-translations.dir.js

```diff
@@ -221,14 +221,16 @@
         }
 
         function saveSentence(sentence) {
+            var lang = sentence.lang === 'unknown' ? '' : $sentence.lang;
```

Comment:
Typo: `$sentence.lang;` should be `sentence.lang;` (no `$`)

Reply:
Thanks!